### PR TITLE
Add dockerfile

### LINKED ISSUE #1: add docker file
This repo should contain a docker file for an image containing all the tools needed to build the Phenoscape KB.

### PR DESCRIPTION
**Pull request by Shalsh23**
Fixes #1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ENV JENA 3.12.0
+ENV BGR 1.4
+ENV KBOT 1.9.1
+
+
+### 2. Get Java and all required system libraries
+RUN apt-get update && apt-get upgrade -y \
+ && apt-get install -y software-properties-common \
+  build-essential git \
+  openjdk-8-jre openjdk-8-jdk
```
Comment (hlapp) [inline Dockerfile:18]: It's best practice to put each piece into its own line. Or everything in one single line, but not half this half that.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+RUN apt-get update \
+    && apt-get install -y make curl git wget tar
```
Comment (hlapp) [inline Dockerfile:21]: This needlessly runs another `apt-get update` (it was just run) and the separate`RUN` instruction introduces another layer in the filesystem, probably needlessly.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+RUN pwd
```
Comment (hlapp) [inline Dockerfile:23]: Any reason for that? It also introduces a layer in the filesystem.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+### 3. Install custom tools
+WORKDIR /tools
+
+RUN pwd
```
Comment (hlapp) [inline Dockerfile:28]: Same as above.

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    && chmod +x robot
+ENV PATH "/tools:$PATH"
+
+
+###### DOSDPTOOLS ######
+RUN wget -nv https://github.com/INCATools/dosdp-tools/releases/download/v$DOSDPVERSION/dosdp-tools-$DOSDPVERSION.tgz \
+&& tar -zxvf dosdp-tools-$DOSDPVERSION.tgz \
+&& mv dosdp-tools-$DOSDPVERSION /tools/dosdp-tools \
+&& wget --no-check-certificate https://raw.githubusercontent.com/INCATools/dead_simple_owl_design_patterns/master/src/simple_pattern_tester.py -O /tools/simple_pattern_tester.py
+ENV PATH "/tools/dosdp-tools/bin:$PATH"
```
Comment (hlapp) [inline Dockerfile:53]: - Should use indentation.
- Why using curl first and now wget?
- Why using `--no-check-certificate`? The SSL cert from raw.githubusercontent.com is valid.

**Comment by balhoff** [comment]
You are right, @hlapp—thanks for your comments!

**Pull request merged**